assessments/guide/security-reviewer.md

```diff
@@ -53,14 +53,37 @@ guidelines for conflict of interest situations are important to prevent:
 - SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
-Hard conflicts makes a reviewer ineligible to review a project.
-A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* Hard conflicts make a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* Soft conflicts are expected for some reviewers, and therefore are fine as long
```

Comment:
```suggestion
* In some cases, one or more reviewers may have a soft conflict.  This may be permissible so long
```
Change is moving plural to singular for clarity.

Reply:
There's been discussion that we really want someone on the review team that has experience using the project, yet that is a soft conflict, so I was trying to word this to make clear that we're actually *encouraging* at least one person with a soft conflict.  I didn't want to make a long section even longer, but maybe it would be good to just say that: 

> Typically, at least one reviewer will have a soft conflict (since the review will benefit from the perspective of someone experienced with the project).  This is permissible so long...

What do you think?

Reply:
hmm, how about:

"It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review."

Reply:
```suggestion
* It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review.
```